api_core/tests/unit/test_page_iterator.py

```diff
@@ -30,15 +30,15 @@ def test_constructor(self):
         parent = mock.sentinel.parent
         item_to_value = mock.sentinel.item_to_value
 
-        page = page_iterator.Page(parent, (1, 2, 3), item_to_value)
+        page = page_iterator.Page(parent, (1, 2, 3), item_to_value, None)
 
         assert page.num_items == 3
         assert page.remaining == 3
         assert page._parent is parent
         assert page._item_to_value is item_to_value
 
     def test___iter__(self):
-        page = page_iterator.Page(None, (), None)
+        page = page_iterator.Page(None, (), None, None)
```

Comment:
Since we don't care about the `raw_page` in this test, we could just omit it (assuming we make it optional) here and below.